template/Berksfile.erb

```diff
@@ -8,3 +8,5 @@ source '<%= s %>'
 <% cookbook.depends.each do |name, cookbook| -%>
 cookbook '<%= name %>', '<%= cookbook.fetch(:version, '>= 0.0.0') %>', <%= cookbook.to_hash %>
 <% end -%>
+
+solver :<%= berkshelf.solver %>
```

Comment:
It doesn't. Anything between `<%=` and `%>` winds up being spit out as a string. So the symbol `:ruby` becomes the string `ruby` and we have to prefix it with a colon for the Berkshelf.